distribution/docker/build.gradle

```diff
@@ -239,9 +252,14 @@ void addBuildDockerImage(Architecture architecture, Variant variant) {
 }
 
 for (final Architecture architecture : Architecture.values()) {
-  for (Variant variant : Variant.values()) {
-    addCopyDockerContextTask(architecture, variant)
-    addBuildDockerImage(architecture, variant)
+  for (final DockerBase base : DockerBase.values()) {
+    for (final boolean oss : [false, true]) {
+      if (oss && base != DockerBase.CENTOS) {
```

Comment:
Let's just add a comment here explaining we only ship a default distribution version of the UBI image.

Reply:
Done.